peerconnection_go_test.go

```diff
@@ -122,6 +123,32 @@ func TestNew_Go(t *testing.T) {
 			}
 		}
 	})
+	t.Run("ICEServers_Copy", func(t *testing.T) {
+		cfg := Configuration{
+			ICEServers: []ICEServer{
+				{
+					URLs: []string{
+						"stun:stun.l.google.com:19302?foo=bar",
+					},
+					Username:   "username",
+					Credential: "password",
+				},
+			},
+		}
+		pc, err := api.NewPeerConnection(cfg)
+		assert.NoError(t, err)
+		assert.NotNil(t, pc)
+
+		cfg.ICEServers[0].Username = util.RandSeq(15)
+		cfg.ICEServers[0].Credential = util.RandSeq(15)
+		cfg.ICEServers[0].URLs[0] = util.RandSeq(15)
+
+		assert.NotEqual(t, pc.configuration.ICEServers[0].Username, cfg.ICEServers[0].Username)
+		assert.NotEqual(t, pc.configuration.ICEServers[0].Credential, cfg.ICEServers[0].Credential)
+		assert.NotEqual(t, pc.configuration.ICEServers[0].URLs, cfg.ICEServers[0].URLs)
```

Comment:
I think it's better to define url, username and password as const string before L127 and assert by `Equal`.

Reply:
Fixed! @at-wat does that capture what you were looking for? Thanks for the quick review :)

Reply:
Yes, that's what I intended!